assets/prototype/domain/eventEditor/containers/queries/useInitQueries.js

```diff
@@ -32,7 +33,7 @@ const useInitQueries = ({ eventId }) => {
 		toaster.error(datetimeError);
 	}
 
-	const datetimes = get(datetimesData, ['datetimes', 'nodes']);
+	const datetimes = pathOr([], ['datetimes', 'nodes'], datetimesData);
```

Comment:
The default value will reduce the boilerplate part:
`Array.isArray(datetimes) &&`